packages/base/src/UI5Element.js

```diff
@@ -712,7 +712,7 @@ class UI5Element extends HTMLElement {
 		// This will be false if the normal event is prevented
 		const normalEventResult = this.dispatchEvent(customEvent);
 
-		// Return false if any of the two events was prevented (its result was false).
+		// Return false if any of the two events were prevented (its result was false).
```

Comment:
Are you sure it must be "were" since "any" is the subject in the sentence (and not "events") and it is singular. One would say "return false if any event was prevented", hence "return false if any of the events was prevented" (and "any" is still the subject). Anyway, not important, just curious :)

Reply:
I read a bit more about this. "Any of" here is plural. Turns out, both forms of the verb are accepted. Use of the singular form is more "formal", plural is more "informal". I'll revert it back since this isn't a grammar error as I thought. See `2. 'any of'` [here](https://www.thefreedictionary.com/any).